recipes/depot.rb

```diff
@@ -6,11 +6,10 @@
 # Copyright (c) 2014 Zühlke, All Rights Reserved.
 
 directory node["software_depot"]
-if node["cache"]
-  node["cache"].each do |e|
-    windev_cache_package e['save_as'] do
-      source e["source"]
-      depot node["software_depot"]
-    end
+
+node.fetch("cache",[]).each do |e|
```

Comment:
nice! I have these if's all over the place and Hash#fetch makes it much better :)